lib/backend/k8s/resources/watcher.go

```diff
@@ -106,6 +106,10 @@ func (crw *k8sWatcherConverter) processK8sEvents() {
 			} else {
 				// We have a valid event, so convert it.
 				e = crw.convertEvent(event)
+				if e == nil {
+					crw.logCxt.Debug("Event converted to a no-op")
+					continue
+				}
```

Comment:
This was causing a panic on line 116, presumably because of the `*e`

Reply:
makes sense.